drivers/xaudio2/audio_driver_xaudio2.cpp

```diff
@@ -48,9 +48,9 @@ Error AudioDriverXAudio2::init() {
 	int latency = Engine::get_singleton()->get_audio_output_latency();
 	buffer_size = closest_power_of_2(latency * mix_rate / 1000);
 
-	samples_in = memnew_arr(int32_t, buffer_size * channels);
+	samples_in = memnew_arr(int32_t, size_t(buffer_size) * channels);
```

Comment:
This was needed to prevent it from becoming:
```suggestion
	samples_in = memnew_arr(int32_t, buffer_size *channels);
```
That's a bug, but it's understandable that it gets confused. `memnew_arr` is a macro and it literally takes a type as first parameter, but the second parameter should be a `size_t` int.